### PR TITLE
Add Active Directory Domain Services dashboards

### PR DESCRIPTION
**Pull request by BinaryFissionGames**
Adds dashboards for the Active Directory Domain Services ops-agent integration.

Dependent on https://github.com/GoogleCloudPlatform/ops-agent/pull/563

### CODE REVIEW

File: dashboards/active-directory-ds/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- These dashboards are based on Google's [Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent).
```
Comment (EvanSimpson) [inline dashboards/active-directory-ds/README.md:1]: Can you add the top level and "Notes" headers here, as the other Readme files do?
e.g. https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/activemq/README.md

File: dashboards/active-directory-ds/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+category: Active Directory
```
Comment (EvanSimpson) [inline dashboards/active-directory-ds/metadata.yaml:1]: I'll note that the format for `metadata.yaml` is still being finalized so there's expectation for it to be included as of now. Feel free to leave or revert, I'll make any necessary adjustments later and outline the expected format in the contribution docs when it's ready.

File: dashboards/active-directory-ds/README.md (hunk 1 above)
Comment (BinaryFissionGames) [reply dashboards/active-directory-ds/README.md:4]: Added these headers, good catch.

File: dashboards/active-directory-ds/metadata.yaml (hunk 2 above)
Comment (BinaryFissionGames) [reply dashboards/active-directory-ds/metadata.yaml]: I've reverted the file for now.

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**